packages/accounts/src/light-account/account.ts

```diff
@@ -108,7 +108,8 @@ export class LightSmartContractAccount<
       encodeFunctionData({
         abi: LightAccountFactoryAbi,
         functionName: "createAccount",
-        args: [await this.owner.getAddress(), this.index],
+        // light account does not support sub-accounts
+        args: [await this.owner.getAddress(), 0n],
```

Comment:
is 0n = 0: number?

Reply:
yeah 0n is a bigint version of 0

Reply:
@avasisht23 can we add a reference link to the light account contract code?